ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -64,8 +64,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-c2s-ipi-disc-priv-fips-f60
-  cron: 36 7 13 2,4,6,8,10,12 *
+- as: aws-c2s-ipi-disc-priv-fips-f360
```

Comment:
The pr is for dropping arm64 test, it looks like we should not touch amd64&multi yaml file.

Reply:
not `drop` but `move` arm to amd

Reply:
but you are changing the test frequency in ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-**4.12__amd64**-nightly-4.12-upgrade-from-stable-4.11.yaml

Reply:
that `Maintenance support ends` at `July 17, 2024`

Reply:
I don't heard of we are planning to drop 4.12 test on amd&multi currently. Per the [reply](https://redhat-internal.slack.com/archives/CF8SMALS1/p1728888379219009?thread_ts=1728886523.478529&cid=CF8SMALS1) from jianwei, and liang's [pr](https://github.com/openshift/release/pull/57738) to reduce the test frequency from f28 to f60, it seems we still need 4.12 test.

cc @jhou1 and @liangxia to confirm.

Reply:
OK, it seems we should keep amd&multi jobs at **f60**, @jhou1 @liangxia please help confirm.

Reply:
https://redhat-internal.slack.com/archives/GH9JHQFKN/p1728613664306739

Reply:
done